.github/workflows/sync.yml

```
@@ -0,0 +1,18 @@
+name: Sync Files
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  convert-test-sync:
+    name: Sync PCL test files
+    runs-on: macos-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: Run GitHub File Sync
+        uses: BetaHuhn/repo-file-sync-action@v1
```

Supply chain security, pin this to ensure we don't run malicious code with our bot token:

```suggestion
        uses: BetaHuhn/repo-file-sync-action@40da4d517e0a744fce9e8aed03e63d6c8bafbdc0 # v1.17.21
```

---

😬